x-pack/legacy/plugins/lens/public/indexpattern_datasource/indexpattern_suggestions.test.tsx

```diff
@@ -839,6 +839,16 @@ describe('IndexPattern Data Source suggestions', () => {
         );
       });
 
+      it('updates references when replacing one metric with another', () => {
+        const initialState = stateWithNonEmptyTables();
+        const suggestions = getDatasourceSuggestionsForField(initialState, '1', {
+          name: 'memory',
+          type: 'number',
+          aggregatable: true,
+          searchable: true,
+        });
+      });
```

Comment:
doesn't this test miss `expect`? 🤔

Reply:
Yes, I actually need to remove this test. I'm not doing the update references step any more.